ctf/src/systems/on_leave/update_gamemode.rs

```diff
@@ -35,9 +35,9 @@ impl<'a> System<'a> for UpdateGameModeOnPlayerLeave {
 			let team = data.teams.get(*ent).unwrap();
 
 			if *team == RED_TEAM {
-				data.gamemode.redteam -= 1;
+				data.gamemode.redteam = std::cmp::max(data.gamemode.redteam - 1, 0);
```

Comment:
See the other comment, left here so that it's easy to apply
```suggestion
				data.gamemode.redteam -= std::cmp::min(data.gamemode.redteam, 1);
```